src/ui/Quickview/QuickviewDocumentIframe.ts

```diff
@@ -7,7 +7,7 @@ export class QuickviewDocumentIframe {
   public el: HTMLElement;
   private iframeElement: HTMLIFrameElement;
 
-  constructor() {
+  constructor(private title?: string) {
```

Comment:
I prefer to not use an optional constructor argument. If we ever need to add a required argument, we will need to introduce a breaking change since required args need to precede optional ones.

Reply:
Suggestion: Inside the `render` method of this class, we are already dealing with an `HTMLDocument`. It's possible that the document already contains a title, and so would could simply use that to set on the iframe.

If this assumption is true (please verify it), then the `title` option/constructor arguments that you are propagating in this PR would not be needed.

Reply:
Good point, I moved this parameter to the `render` function instead.